lib/pbench/server/api/__init__.py

```diff
@@ -92,15 +92,36 @@ def _corsify_actual_response(response):
     return response
 
 
-class GraphQL(Resource):
+class CorsOptionsRequest:
+    def __init__(self, options_exception_msg):
```

Comment:
The way your classes are set up and named, the error message could have just used the class name in a template string without the extra parameter, but it doesn't really hurt.